content/en/agent/guide/python-3.md

```diff
@@ -9,6 +9,31 @@ This guide provides information and best practices on migrating checks between P
 
 To provide flexibility in allowing code to run multiple on versions of the Agent, this guide focuses on retaining backwards compatibility.
 
+## Agent configuration
+
+Starting with v6.14.0, the Agent integrates both Python 2 and Python 3 runtimes. This means that checks can be run with either Python 2 or Python 3, depending on the Agent configuration.
+
+By default, the Agent v6 will use the Python 2 runtime. To switch to the Python 3 runtime, set the `python_version` configuration option in the `datadog.yaml` configuration file:
```

Comment:
I think we should do a numbered bullet point list, like:

1. Set the ...
2. [Restart the Agent](/agent/guide/agent-commands/?tab=agentv6#restart-the-agent).